elliott/elliottlib/cli/create_cli.py

```diff
@@ -128,31 +175,31 @@ async def create_cli(
         $ elliott -g openshift-4.14 create --art-advisory-key image --date 2018-Mar-05 --yes
     """
     runtime.initialize()
-
     et_data = runtime.get_errata_config()
 
-    if sum(map(bool, [date, batch_id])) != 1:
```

Comment:
why remove this?

Reply:
ah i see it got moved.. but why move?

Reply:
apologies - added in https://github.com/openshift-eng/art-tools/pull/1573/commits/588bcdd0118448bef4189750bf0d00936f20122d